examples/platforms/samr21/samr21x18a.ld

```diff
@@ -0,0 +1,166 @@
+/**
```

Comment:
could we change to openthread's license? if not, maybe we can move this ld script to the third party.

Reply:
I will move ld script and some other headers to third party.